Docs - Added clarity to location of download folder for Home Assistan…

IronTooch opened the pull request:
## Proposed change
In the previous documentation, it's not clear where that folder should exist or live. This is a pure docs update that adds clarity to usage of Downloader Add-on for folder locations, by including root path for Home Assistant OS installs



## Type of change
Adds clarity to usage of Downloader Add-on for folder locations, by including root path for Home Assistant OS installs

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

- **Documentation**
	- Updated the documentation for the Downloader integration to clarify file path handling and provide a more accurate example for users.
	- Introduced new instructions regarding user input for folder names to enhance understanding.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

IronTooch committed:
Docs - Added clarity to location of download folder for Home Assistant OS

In the previous documentation, it's not clear where that folder should exist or live.

IronTooch committed:
fix - Corrected spelling of `initial`

c0ffeeca7 commented on source/_integrations/downloader.markdown:
````diff
@@ -60,3 +60,7 @@ Along with the event the following payload parameters are available:
       message: "{{trigger.event.data.filename}} download failed"
       title: "Download Failed"
  ```
+
````
@IronTooch, the file path is discussed in line 19. Please move your comment there.

IronTooch replied:
Moved as suggested.

IronTooch committed:
Docs - Moved Home Assistant OS example to same line as configuration folder details. Removed "See Location of Folder"

Removed "Location of Folder" header and just placed example with configuration of Home Assistant OS with the other config items.

IronTooch committed:
Merge branch 'current' into patch-1

c0ffeeca7 commented on source/_integrations/downloader.markdown:
```diff
@@ -16,7 +16,7 @@ The **Downloader** {% term integration %} provides an action to download files.
 {% include integrations/config_flow.md %}
 
-If the path is not absolute, it’s assumed to be relative to the Home Assistant configuration directory (for example, .homeassistant/downloads).
+If the path is not absolute, it’s assumed to be relative to the Home Assistant configuration directory (for example, `.homeassistant/downloads`). For Home Assistant OS, the Downloader extension is based off the initial folder `/root/homeassistant/`. To that effect, if you have a folder called `/root/homeassistant/my_download_folder`, you would enter simply `my_download_folder` under the prompt fo ...
```
```suggestion
If the path is not absolute, it’s assumed to be relative to the Home Assistant configuration directory (for example, `.homeassistant/downloads`). When prompted to enter a folder name, only enter the folder name, not the path. So if you have a folder called `.homeassistant/my_download_folder`, when prompted to **Select a location to get to store downloads**, enter `my_download_folder`. Home Assistant checks if the directory exists.
```

c0ffeeca7 committed:
rephrase

c0ffeeca7 reviewed (dismissed):
Thank you, @IronTooch :+1:

c0ffeeca7 commented on source/_integrations/downloader.markdown:
```diff
@@ -16,7 +16,7 @@ The **Downloader** {% term integration %} provides an action to download files.
 {% include integrations/config_flow.md %}
 
-If the path is not absolute, it’s assumed to be relative to the Home Assistant configuration directory (for example, .homeassistant/downloads).
+If the path is not absolute, it’s assumed to be relative to the Home Assistant configuration directory (for example, `.homeassistant/downloads`). When prompted to enter a folder name, only enter the folder name, not the path. So if you have a folder called `.homeassistant/my_download_folder`, when prompted to **Select a location to get to store downloads**, enter `my_download_folder`. Home Assist ...
```
```suggestion
If the path is not absolute, it’s assumed to be relative to the Home Assistant configuration directory (for example, `/config/downloads`). When prompted to enter a folder name, only enter the folder name, not the path. So if you have a folder called `/config/my_download_folder`, when prompted to **Select a location to get to store downloads**, enter `my_download_folder`. Home Assistant checks if the directory exists.
```

c0ffeeca7 committed:
change  homeassistant folder to config, assuming HA OS

c0ffeeca7 commented on source/_integrations/downloader.markdown:
```diff
@@ -16,7 +16,7 @@ The **Downloader** {% term integration %} provides an action to download files.
 {% include integrations/config_flow.md %}
 
-If the path is not absolute, it’s assumed to be relative to the Home Assistant configuration directory (for example, .homeassistant/downloads).
+If the path is not absolute, it’s assumed to be relative to the Home Assistant configuration directory (for example, `/config/downloads`). When prompted to enter a folder name, only enter the folder name, not the path. So if you have a folder called `/config/my_download_folder`, when prompted to **Select a location to get to store downloads**, enter `my_download_folder`. Home Assistant checks if  ...
```
```suggestion
If the path is not absolute, it’s assumed to be relative to the Home Assistant configuration directory (for example, `/config/downloads`). So if you have a folder called `/config/my_download_folder`, when prompted to **Select a location to get to store downloads**, enter `my_download_folder`. Home Assistant checks if the directory exists.
```

c0ffeeca7 committed:
Update source/_integrations/downloader.markdown
